subt/test_artf_node.py

```diff
@@ -58,4 +60,53 @@ def test_merge_two_results(self):
         self.assertEqual(checked_result, expected_result)
 
 
+if __name__ == "__main__":
```

Comment:
So far we have been using `test_<something>.py` to only run unittests. This kind of tests were usually in the `<something>.py` file. I am not saying we cannot change the convention, just that we should do it consciously.

Reply:
well this was supposed to be "extended test" (regression test?) based on previously logged that, but in meantime I found out that I do not have necessary inputs to do that (and the logfiles twice changed recently). I may move it, thanks.

Reply:
OK, moved but ignore it at the moment - I need to log the detector outputs stream so that I can analyze the "fusion" and then the depth/xyz calculation